yt_dlp/extractor/dailywire.py

```diff
@@ -0,0 +1,114 @@
+from .common import InfoExtractor
+from ..utils import (
+    determine_ext,
+    float_or_none,
+    join_nonempty,
+    traverse_obj,
+    url_or_none,
+)
+
+
+class DailyWireBaseIE(InfoExtractor):
+    _JSON_PATH = {
+        'episode': ('props', 'pageProps', 'episodeData', 'episode'),
+        'videos': ('props', 'pageProps', 'videoData', 'video'),
+        'podcasts': ('props', 'pageProps', 'episode'),
+    }
+
+    def _get_json(self, url):
+        sites_type, slug = self._match_valid_url(url).group('sites_type', 'id')
+        json_data = self._search_nextjs_data(self._download_webpage(url, slug), slug)
+        return slug, traverse_obj(json_data, self._JSON_PATH[sites_type])
+
+
+class DailyWireIE(DailyWireBaseIE):
+    _VALID_URL = r'https?://(?:www\.)dailywire(?:\.com)/(?P<sites_type>episode|videos)/(?P<id>[\w-]+)'
+    _TESTS = [{
+        'url': 'https://www.dailywire.com/episode/1-fauci',
+        'info_dict': {
+            'id': 'ckzsl50xnqpy30850in3v4bu7',
+            'ext': 'mp4',
+            'display_id': '1-fauci',
+            'title': '1. Fauci',
+            'description': 'Lee explores the history of the politics surrounding Covid-19, and reveals shocking details about both Anthony Fauci\'s role in hiding critical information from the American people as well as worrisome unanswered questions surrounding the origins of the virus.',
```

Comment:
Use md5

tip: If you run a wrong test-case, it will tell you how to fix itself

Reply:
Fixed